euphonic/spectra/base.py

```diff
@@ -675,14 +716,18 @@ def broaden(self: T, x_width,
             y_broadened = ureg.Quantity(y_broadened, units=self.y_data_unit)
 
         elif isinstance(x_width, Callable):
-            self.assert_regular_bins(message=(
-                'Broadening with convolution requires a regular sampling grid.'
-            ))
+            self.assert_regular_bins(
+                message=(
+                    "Broadening with convolution requires a "
+                    "regular sampling grid."
+                ),
+                restrict_range=False,
+            )
             y_broadened = variable_width_broadening(
-                self.get_bin_edges(),
+                self.get_bin_edges(restrict_range=False),
```

Comment:
Does the fact that this can change bins need to be documented in this docstring directly?

Reply:
That makes sense!